### ISSUE TITLE
OSM notes in Overpass turbo

### ISSUE THREAD

**Issue by rmikke** [tyrasd/overpass-turbo#216]
I have searched for a way to display OSM notes (like this: http://www.openstreetmap.org/note/476576 ) and maybe I'm really bad at searching, but all I have found is the possibility for it in Overpass API ( http://wiki.openstreetmap.org/wiki/API_v0.6#Map_Notes_API ) and it seems there is no way to use it Overpass Turbo.

It would be useful to have notes and whatever I'm interested in updating right now, on one map, for planning where to go for the best effect for OSM.

**Comment by HolgerJeromin** [comment]
Just to clarify: You found the documentation of the main OSM API, not Overpass API. 
Still it would be possible to integrate in Overpass Turbo.

**Comment by mmd-osm** [comment]
Right. Overpass API doesn't support OSM notes at all. That's only available on the main OSM API, which is something completely different.

**Comment by rmikke** [comment]
OK, you are right of course, it's OSM API.
Still, it would be good to find OSM notes with Overpass.
